upcloud_api/cloud_manager/base.py

```diff
@@ -5,6 +5,7 @@
 
 from upcloud_api import UpCloudAPIError
 
+version = '0.4.6'
```

Comment:
setup.py has the version already, could you get it from there? or put the version in some other file and get version in both files from there? :)

Reply:
Tried getting it from the setup.py. Would need some hacky solution though, since python and package path etc problems. Was also thinking of getting the version from git, but that's not good either.

Reply:
Would want to have it generated programmatically though, since in future I would want conventional commits to generate release tags and changelog. Though if it is in some sort of "client_description.py" file, then it might be updated same way as those are.

Reply:
having version in client_description.py or info.py or something like that sounds good to me

Reply:
or even version.py as in Ruby world :D 

```
# version.rb
module ApiV1
  VERSION = "0.0.1"
end
```